content/en/tracing/guide/latency_investigator.md

```diff
@@ -0,0 +1,120 @@
+---
+title: APM Investigator
+private: true
+further_reading:
+- link: "/tracing/"
+  tag: "Documentation"
+  text: "Datadog APM"
+---
+
+{{< callout url="https://www.datadoghq.com/support/" header="Request access to the Preview!" >}}
+APM Investigator is in Preview. To request access, fill out this form.
+{{< /callout >}}
+
+## Overview
+
+The APM Investigator diagnoses and resolves application latency issues through a guided, step-by-step workflow. It consolidates analysis tools in a single interface to accelerate root cause identification.
+
+{{< img src="tracing/guide/apm_investigator/apm_investigator.png" alt="APM Investigator UI" style="width:90%;">}}
+
+## Key capabilities
+
+The APM Investigator helps you:
+
+- **Investigate slow request clusters**: Select problematic requests directly from the latency scatter plot.
+- **Identify the source of latency**: Determine whether latency originates from your service, a downstream dependency, databases, or third-party APIs.
```

Comment:
The switch in this list from singular to plurals gave me pause for a second, but if it's intentional, ignore me!